[doc] broken links

Link verification failed in aggregate reports (internal) https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1172661&view=logs&j=d41e9505-d81b-5b3c-5b50-be71bc9b0ccb&t=9ff5a92d-14ef-5b48-a596-30a735a1a817

```
Summary of broken links:
'file:///D:/a/1/s/sdk/monitor/monitor-opentelemetry-exporter/README.md' has 1 broken link(s):
  https://github.com/open-telemetry/opentelemetry-js/blob/master/getting-started/README.md
'file:///D:/a/1/s/documentation/Bundling.md' has 1 broken link(s):
  https://parceljs.org/typeScript.html
'file:///D:/a/1/s/sdk/iot/iot-modelsrepository/README.md' has 1 broken link(s):
  https://github.com/Azure/azure-iot-sdk-node/tree/master/device/samples/pnp/readme.md
##[error]Checked 2446 links with 3 page(s) broken.
```
new valid links:

- open telemetry js: https://opentelemetry.io/docs/js/getting-started/
- parcel typescript: https://parceljs.org/languages/typescript/
- pnp: https://github.com/Azure/azure-iot-sdk-node/tree/master/device/samples#plug-and-play-examples

--- a/documentation/Bundling.md
+++ b/documentation/Bundling.md
@@ -443,7 +443,7 @@ Now let's create a very basic [tsconfig.json](https://www.typescriptlang.org/doc
 }
 ```
 
-For more information on using Parcel with TypeScript, check out the TypeScript guide in Parcel's documentation: https://parceljs.org/typeScript.html
+For more information on using Parcel with TypeScript, check out the [TypeScript guide in Parcel's documentation](https://parceljs.org/languages/typescript/)
 
 Similar to our JS example above, let's create an `index.ts` file that imports from `@azure/storage-blob`:
 

--- a/sdk/iot/iot-modelsrepository/README.md
+++ b/sdk/iot/iot-modelsrepository/README.md
@@ -156,7 +156,7 @@ console.log(fullyQualifiedModelPath);
 
 - Review the [DTDL Spec](https://docs.microsoft.com/azure/iot-pnp/concepts-model-parser).
 - Understand the [Device Models Repository](https://devicemodels.azure.com/).
-- Code a IoT Plug and Play 'Device' using the [Azure IoT SDK for Node](https://github.com/Azure/azure-iot-sdk-node/tree/master/device/samples/pnp/readme.md).  
+- Code a IoT Plug and Play 'Device' using the [Azure IoT SDK for Node](https://github.com/Azure/azure-iot-sdk-node/tree/master/device/samples#plug-and-play-examples).  
 
 # Related projects
 

--- a/sdk/monitor/monitor-opentelemetry-exporter/README.md
+++ b/sdk/monitor/monitor-opentelemetry-exporter/README.md
@@ -4,7 +4,7 @@
 
 ## Getting started
 
-This exporter package assumes your application is [already instrumented](https://github.com/open-telemetry/opentelemetry-js/blob/master/getting-started/README.md) with the OpenTelemetry SDK. Once you are ready to export OpenTelemetry data, you can add this exporter to your application.
+This exporter package assumes your application is [already instrumented](https://opentelemetry.io/docs/js/getting-started/) with the OpenTelemetry SDK. Once you are ready to export OpenTelemetry data, you can add this exporter to your application.
 
 ### Install the package
 

